### PR TITLE
json_api adapter replacement

### PR DESCRIPTION
**Pull request by ldlsegovia**
Use `:json` instead of `:json_api` adpater

Collection example:

image

Resource example:

image

### CODE REVIEW

File: lib/power_api/generator_helper/swagger_helper.rb [hunk 1]
```diff
@@ -268,7 +256,7 @@ def spec_tpl_index
         "response '200', '#{resource.plural_titleized} retrieved' do",
           "schema('$ref' => '#/definitions/#{resource.plural}_collection')\n",
           "run_test! do |response|",
-            "expect(JSON.parse(response.body)['data'].count).to eq(expected_collection_count)",
+            "expect(JSON.parse(response.body)['#{resource.plural}'].count).to eq(expected_collection_count)",
```
Comment (monkeyci) [inline lib/power_api/generator_helper/swagger_helper.rb:259]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is too long. [109/100]

File: spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
   describe "#resource_attributes" do
     let(:expected_attributes) do
       [
+{ name: :id, type: :integer, swagger_type: :integer, example: kind_of(Integer), required: false },
```
Comment (monkeyci) [inline spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.

File: lib/power_api/generator_helper/active_record_resource.rb [hunk 3]
```diff
@@ -22,7 +22,10 @@ def resource_name=(value)
 
   def resource_attributes=(collection)
     attributes = format_attributes(collection)
-    raise PowerApi::GeneratorError.new("at least one attribute must be added") if attributes.none?
+
```
Comment (monkeyci) [inline lib/power_api/generator_helper/active_record_resource.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing whitespace detected.

File: lib/power_api/generator_helper/swagger_helper.rb (hunk 1 above)
Comment (ldlsegovia) [reply lib/power_api/generator_helper/swagger_helper.rb:259]: me gusta como está jojoo

File: README.md [hunk 4]
```diff
@@ -522,7 +536,7 @@ after doing this you will get:
       end
 
       let(:attributes) do
-        JSON.parse(response.body)['data']['attributes'].symbolize_keys
+        JSON.parse(response.body).symbolize_keys
```
Comment (difernandez) [inline README.md]: There should be a `['blog']` here I think

File: spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb [hunk 5]
```diff
@@ -50,10 +54,21 @@ def perform
...
+    context "with included id" do
+      let(:include_id) { true }
+      let(:expected_attributes) do
+        [
+          { name: :id, type: :integer, swagger_type: :integer, example: kind_of(Integer), required: false },
+          { name: :title, type: :string, swagger_type: :string, example: "'Some title'", required: true },
+          { name: :body, type: :text, swagger_type: :string, example: "'Some body'", required: true }
+        ]
+      end
+    end
```
Comment (difernandez) [inline spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb]: - This context doesn't have an example
- Shouldn't it be `required: true` in the `id` line?

**Review by difernandez — CHANGES_REQUESTED** [review]

File: spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb (hunk 5 above)
Comment (ldlsegovia) [reply spec/dummy/spec/support/shared_examples/active_record_resource_atrributes.rb]: No, id column should be treated like special attribute

**Review by difernandez — APPROVED** [review]

**Pull request merged**